tasks/pkg-redhat/install-latest.yml

```diff
@@ -1,7 +1,16 @@
 ---
+- name: Set installation target for latest
+  set_fact:
+    # Apply a version cap if required, otherwise just target the agent package
+    datadog_agent_target: >-
+      {{ datadog_agent_flavor }}
+      {% if datadog_agent_max_minor_version is defined %}
+       < 1:{{ agent_datadog_agent_major_version }}.{{ datadog_agent_max_minor_version + 1 }}.0-1
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏ Wouldn't this always end up targeting `7.52.0`?

Reply:
It might not be obvious with all the templating, but this actually sets the target to `"datadog-agent < 1:7.52.0-1"` as suggested by @bkabrda above. In my local tests on Centos 6 this results in the installation of 7.51.1 which is what we want.

Reply:
Oh, as you say that now it seems more obvious 👍